webextensions/api/cookies.json

```diff
@@ -615,7 +615,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": true
+                  "version_added": "76"
```

Comment:
While we're here, should we add a more meaningful description, like ``"Defaults to `unspecified`"``?

> This CL changes the default value to "unspecified", which corresponds to a cookie set without a SameSite attribute. ([Source](https://chromiumdash.appspot.com/commit/218d4eae63313b8fcf80f4befab679d1a200c57b))